RS.Fritz.Manager.Domain/HttpServices/HttpsNonValGetService.cs

```diff
@@ -0,0 +1,47 @@
+﻿namespace RS.Fritz.Manager.Domain
+{
+    using System;
+    using System.Net;
+    using System.Net.Http;
+    using System.ServiceModel;
+    using System.Threading.Tasks;
+
+    public sealed class HttpsNonValGetService : IHttpGetService
+    {
+        private readonly IHttpClientFactory httpClientFactory;
+        private readonly EndpointAddress endpointAddress;
+        private readonly NetworkCredential networkCredential;
+        private readonly FritzServiceEndpointConfiguration endpointConfiguration;
+
+        public HttpsNonValGetService(FritzServiceEndpointConfiguration endpointConfiguration, EndpointAddress remoteAddress, NetworkCredential networkCredential, IHttpClientFactory httpClientFactory)
+        {
+            this.endpointAddress = remoteAddress;
+            this.networkCredential = networkCredential;
+            this.endpointConfiguration = endpointConfiguration;
+            this.httpClientFactory = httpClientFactory;
+        }
+
+        public async Task<string> GetHttpResponseAsync()
+        {
+            // Configuration is done in App.xaml.cs
+            HttpClient httpClient = httpClientFactory.CreateClient(Constants.NonValidatingHttpsClientName);
+
+            HttpResponseMessage? result = await httpClient.GetAsync(endpointAddress.Uri);
+
+            string content = string.Empty;
+            if (result.StatusCode == HttpStatusCode.OK)
```

Comment:
In case we send a bad request or receive an error we are not notified. I would replace this code with:
result.EnsureSuccessStatusCode();

return await httpClient.GetStringAsync(endpointAddress.Uri);